File: php/Assignment 4/SET A/a1.php
<html>

<body>
    <form method='POST'>
        Enter radius:
        <input type='text' name='radius'><br>
        Enter height:
        <input type='text' name='height'><br>
        <pre><input type='submit' name='submit' value='Area.'>  <input type='reset' name='reset' value='reset'>
</form>
<?php
$r = $_POST['radius'];
$h = $_POST['height'];
define('PPI', '3.1412');
interface fun
{
    function area($r, $h);
    function volume($r, $h);
}
class cylinder implements fun
{
    function area($r, $h)
    {
        $area = (2 * PPI * $r * ($r + $h));
        echo "<center><h3>The area of cylinder is:     $area</center></h3>";
    }
    function volume($r, $h)
    {
        $v = (PPI * $r * $r * $h);
        echo "<center><h3>The volume of cylinder is:     $v</center></h3>";
    }
}
$o = new cylinder;
$o->area($r, $h);
$o->volume($r, $h);
?>
</body>
</html>

File: cpp/Assignment 2/SET A/a1.cpp
#include<iostream>
using namespace std;
class Cylinder{
    private:
        int radius, height;
        float area1;
    public:
        void aradius(){
            cout<< endl <<"Enter Radius";
            cin>>radius;
        }
        void aheight(){
            cout<< endl <<"Enter Height" << endl;
            cin>>height;
        }
        void volume(){
            float pi = 3.14;
            int r = radius , h = height;
            float volume = pi * (r * r) * h;  
            cout<< endl <<volume << endl ;
        }
        void area(){
            float pi = 3.14;
            int r = radius , h = height;
            area1 = 2 * pi * r * h + 2 * pi * r * r;
            cout<< endl <<area1;    
        }
};
int main(){
    Cylinder Obj1, Obj2;
    Obj1.aradius();
    Obj1.aheight();
    Obj1.volume();
    Obj1.area();    
    Obj2.aradius();
    Obj2.aheight();
    Obj2.volume();
    Obj2.area();    
    return 0;
}